### PR TITLE
Cloud homepage

### PR DESCRIPTION
**Pull request by Deflaimun**
Creates a new cloud homepage

Preview:
https://deploy-preview-762--redpanda-documentation.netlify.app/
https://deploy-preview-762--redpanda-documentation.netlify.app/docs/home/core/
https://deploy-preview-762--redpanda-documentation.netlify.app/docs/home/cloud/

### CODE REVIEW

File: docs/home/HomepageFeatures.js [hunk 1]
```diff
@@ -4,71 +4,30 @@ import styles from './HomepageFeatures.module.css';
...
 const FeatureList = [
   {
-    title: '60-Second Guides',
-    image: require('../../static/img/quickstart.png').default,
+    title: 'Core',
+    image: require('../../static/img/panda.png').default,
     description: (
       <>
-        Try out Redpanda.
+        ccccccccccccccccccccccccccc
```
Comment (Deflaimun) [inline docs/home/HomepageFeatures.js:11]: Please advice for new intro

File: docs/home/HomepageFeatures.js [hunk 2]
```diff
@@ -4,71 +4,30 @@ import styles from './HomepageFeatures.module.css';
...
   },
   {
-    title: 'Deploying for Production',
-    image: require('../../static/img/deployment.png').default,
+    title: 'Cloud',
+    image: require('../../static/img/cloud.png').default,
     description: (
       <>
-        Install and configure Redpanda.
+        ccccccccccccccccccccccccccc
```
Comment (Deflaimun) [inline docs/home/HomepageFeatures.js:21]: Please advice for new intro

File: docs/home/cloud/cloud.js [hunk 3]
```diff
@@ -0,0 +1,101 @@
...
+import clsx from 'clsx';
+import styles from './cloud.module.css';
+
+const FeatureList = [
+  {
+    title: 'Managed Connectors',
+    image: require('../../../static/img/onboard.png').default,
+    description: (
+      <>
+        xxxxxxxxxxxxxxxxxxx
```
Comment (Deflaimun) [inline docs/home/cloud/cloud.js:11]: Please advice for intro to managed connectors

File: docs/home/cloud/cloud.js [hunk 4]
```diff
@@ -0,0 +1,101 @@
...
+      </>
+    ),
+    url: '/docs/cloud/managed-connectors/',
+  },
+  {
+    title: 'Dedicated vs BYOC page',
+    image: require('../../../static/img/console-icon.png').default,
+    description: (
+      <>
+        xxxxxxxxxxxxxxxxxxxxxxxxxxxx
```
Comment (Deflaimun) [inline docs/home/cloud/cloud.js:21]: Please advice for intro to dedicated vs byoc page

File: docs/home/cloud/cloud.js [hunk 5]
```diff
@@ -0,0 +1,101 @@
...
+      </>
+    ),
+    url: '/docs/cloud/dedicated-byoc',
+  },
+  {
+    title: 'Creating a dedicated cluster',
+    image: require('../../../static/img/deploy.png').default,
+    description: (
+      <>
+        xxxxxxxxxxxxxxxxxxxxxxxxxxx
```
Comment (Deflaimun) [inline docs/home/cloud/cloud.js:31]: Please advice for creating a dedicated cluster intro

File: docs/home/cloud/cloud.js [hunk 6]
```diff
@@ -0,0 +1,101 @@
...
+      </>
+    ),
+    url: '/docs/cloud/create-dedicated-cloud-cluster-aws',
+  },
+  {
+    title: 'Creating a BYOC cluster',
+    image: require('../../../static/img/multi-cluster.png').default,
+    description: (
+      <>
+        xxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxx
```
Comment (Deflaimun) [inline docs/home/cloud/cloud.js:41]: Please advice for creating a byoc intro

File: docs/home/cloud/cloud.js [hunk 7]
```diff
@@ -0,0 +1,101 @@
...
+      </>
+    ),
+    url: '/docs/cloud/create-byoc-cluster-aws',
+  },
+  {
+    title: 'VPC Peering',
+    image: require('../../../static/img/cloud.png').default,
+    description: (
+      <>
+        xxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxx
```
Comment (Deflaimun) [inline docs/home/cloud/cloud.js:51]: Please advice for vpc peering

File: docs/home/HomepageFeatures.js [hunk 8]
```diff
@@ -4,71 +4,30 @@ import styles from './HomepageFeatures.module.css';
 
 const FeatureList = [
   {
-    title: '60-Second Guides',
-    image: require('../../static/img/quickstart.png').default,
+    title: 'Core',
```
Comment (Feediver1) [inline docs/home/HomepageFeatures.js:7]: @Deflaimun Let's use:
Redpanda Platform
Redpanda Cloud

**Comment by Deflaimun** [comment]
Creating a dedicated cluster:
"Deploy a cluster in Redpanda's VPC"
Creating a BYOC cluster:
"Deploy a cluster in your own VPC"
Dedicated vs. BYOC:
 "Understand the differences between deploying in Redpanda's dedicated cloud and deploying in your own VPC"
Managed Connectors:
"Export data from topics to external systems using managed connectors"
VPC Peering:
"Create a networking connection between your VPC and Redpanda's VPC" (edited)

**Comment by AnneLeeperRP** [comment]
LGTM!

**Pull request merged**